### DISCUSSION TITLE
Location of project file

### DISCUSSION THREAD

**Discussion by iso1048** [marcusolsson/obsidian-projects#763] [Help] [answered]
Hi. 

I'm new to obsidian and have just installed this plugin to create a library to track books - I've called this project "Book Library". 

1) This is probably a noob question but where is the markdown file of the "Book Library" stored? 
2) Is there a plan to allow the data type of a column to be changed e.g. number, tags, text?

Thanks in advance.

**Comment by Acylation** [comment]
The file path is determined by the settings of each project, that can be configured when creating or, in the three-dots dropdown besides project selector.

<img width="221" alt="image" src="https://github.com/marcusolsson/obsidian-projects/assets/73122375/853139c0-9cdb-4ddd-b497-58ab296dd61d">
<img width="350" alt="image" src="https://github.com/marcusolsson/obsidian-projects/assets/73122375/2c0320ea-e19f-4d18-9218-229a9d0eadb4">
  
Type casting will definately be supported. I partly implemented on field creation and will work on this soon. You can check [active issues](https://github.com/marcusolsson/obsidian-projects/issues?q=is%3Aissue+is%3Aopen+label%3Alifecycle%2Factive) and the [next milestone](https://github.com/marcusolsson/obsidian-projects/milestones) to see what's going on.

https://github.com/marcusolsson/obsidian-projects/blob/490493eba4fc5c7060ee1230d247a857fb1b1e4a/src/ui/modals/components/CreateField.svelte#L63-L113

**Reply by iso1048** [reply]
Thanks for your reply. Sorry I think I still need to clarify something - is the project view saved as a separate markdown file, or is the project view just re-rendered each time from the data selected in the settings? Thank you!

**Reply by Acylation** [reply, marked as answer]
Projects examines the target folder (or tags / dataview query according to your project config) to gather the YAML data, and will get dymatically updated when you edit the note properties directly or by other methods. Currently it doesn't own a separated file mode, but if you feel this useful, please feel free to submit a [feature request](https://github.com/marcusolsson/obsidian-projects/issues/new?labels=kind%2Ffeature%2Cneeds-triage&projects=&template=feature-request.yml) to describe your case.

Btw if you're interested in exporting the projects data as text/table files there is an existing FR #318 / discussion #489 .

**Reply by iso1048** [reply]
That helps to clear things up. Thank you very much for taking the time to help, it is much appreciated :)
